extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -707,7 +707,7 @@ export class AmpStoryPage extends AMP.BaseElement {
         mediaEl.addEventListener('error', resolve, true /* useCapture */);
       });
     });
-    return Promise.all(mediaPromises).then(() => this.markPageAsLoaded_());
+    return Promise.all(mediaPromises);
```

Comment:
Do we really need to change this? We'd need to think of the repercussions it could have